PUBLIC_DOCS.md

```diff
@@ -0,0 +1,69 @@
+## Endpoints
+
+Return the Mode of the Light Client
+
+```bash
+curl -s localhost:7000/v1/mode
+```
+
+Result:
+
+```json
+"LightClient"
+```
+
+Returns the latest block 
+
+```bash
+curl -s localhost:7000/v1/latest_block
+```
+
+Result:
+
+```json
+{"latest_block":5}
+```
+
+Given a block number (as _(hexa-)_ decimal number), return confidence obtained by the light client for this block:
+
+```bash
+curl -s localhost:7000/v1/confidence/ <block-number>
+```
+
+Result:
+
+```json
+{
+    "number": 223,
+    "confidence": 99.90234375,
+    "serialisedConfidence": "958776730446"
+}
+```
+
+>  `serialisedConfidence` is calculated as: 
+> `blockNumber << 32 | int32(confidence * 10 ** 7)`, where confidence is represented out of 10 ** 9.
+
+
+Given a block number (as _(hexa-)_ decimal number), return the content of the data if specified in config in hex string format
```

Comment:
The sentence might be a bit confusing to a new user. Can we rephrase?

Reply:
Fixed